File: C/Sully.c
#include <stdio.h>
#include <stdlib.h>
#define CODE "#include <stdio.h>%1$c#include <stdlib.h>%1$c#define CODE %3$c%4$s%3$c%1$c%1$cint main(void) {%1$c%2$cint i = %5$d;%1$c%2$cchar filename[50], execname[50], cmd[120];%1$c%1$c%2$cif (i <= 0) return 0;%1$c%2$ci--;%1$c%2$csprintf(filename, %3$cSully_%%d.c%3$c, i);%1$c%2$csprintf(execname, %3$cSully_%%d%3$c, i);%1$c%2$cFILE *file = fopen(filename, %3$cw%3$c);%1$c%2$cif (!file) return 1;%1$c%2$cfprintf(file, CODE, 10, 9, 34, CODE, i);%1$c%2$cfclose(file);%1$c%1$c%2$csprintf(cmd, %3$cgcc -Wall -Wextra -Werror -o %%s %%s && ./%%s%3$c, execname, filename, execname);%1$c%2$csystem(cmd);%1$c%2$creturn 0;%1$c}%1$c"

int main(void) {
	int i = 5;
	char filename[50], execname[50], cmd[120];

	if (i <= 0) return 0;
	i--;
	sprintf(filename, "Sully_%d.c", i);
	sprintf(execname, "Sully_%d", i);
	FILE *file = fopen(filename, "w");
	if (!file) return 1;
	fprintf(file, CODE, 10, 9, 34, CODE, i);
	fclose(file);

	sprintf(cmd, "gcc -Wall -Wextra -Werror -o %s %s && ./%s", execname, filename, execname);
	system(cmd);
	return 0;
}

File: py/Sully.py
import os
def happpy_sully():
	i = 5
	if i <= 0: return 0
	i -= 1
	code = "import os{2}def happpy_sully():{2}{1}i = {4}{2}{1}if i <= 0: return 0{2}{1}i -= 1{2}{1}code = {3}{0}{3}{2}{1}filename = f'Sully_{{i}}.py'{2}{1}with open(filename, {3}w{3}) as f:{2}{1}{1}f.write(code.format(code, chr(9), chr(10), chr(34), i)){2}{1}os.system(f'python3 {{filename}}'){2}{2}happpy_sully(){2}"
	filename = f'Sully_{i}.py'
	with open(filename, "w") as f:
		f.write(code.format(code, chr(9), chr(10), chr(34), i))
	os.system(f'python3 {filename}')

happpy_sully()
